optuna/integration/fastai.py

```diff
@@ -1,15 +1,14 @@
+from typing import Any
+
 import optuna
 from optuna._imports import try_import
-from optuna import type_checking
-
-if type_checking.TYPE_CHECKING:
-    from typing import Any  # NOQA
 
 with try_import() as _imports:
     from fastai.basic_train import Learner  # NOQA
     from fastai.callbacks import TrackerCallback
 
 if not _imports.is_successful():
+    Learner = object  # NOQA
```

Comment:
These will be [rendered as `object` in the docs](https://54001-122299416-gh.circle-artifacts.com/0/docs/build/html/reference/generated/optuna.integration.FastAIPruningCallback.html#optuna.integration.FastAIPruningCallback) which is somewhat misleading. Do you think it's possible to omit these `object` assignments altogether and and modify the type hints to in this case `"Learner"` instead fo `Learner`? We do that in several other integration modules e.g. https://github.com/optuna/optuna/blob/master/optuna/integration/chainer.py#L49.

Reply:
Sure. Thank you for reviewing and kindly showing an example.